examples/geometry_from_files.py

```diff
@@ -1,8 +1,12 @@
 import flow360 as fl
 from flow360.component.geometry import Geometry
-from flow360.examples import Octahedron
+from flow360.examples import CylinderGeometry
 
-geometry = Geometry.from_file(Octahedron.geometry, name="octahedron")
+fl.Env.dev.active()
+
+geometry = Geometry.from_file(
+    CylinderGeometry.geometry, name="testing-cylinder-geometry", solver_version="release-24.2.2"
```

Comment:
I think geometry now works on prod. Can we remove dev and exact solver version (24.2.2) ?

Reply:
removed